Description: several simple warning fix
Author: Nicolas HOUDELOT (nicolas@demosdebs.org)
Last-Update: 2020-04-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/luisita.cpp
+++ b/src/luisita.cpp
@@ -4,6 +4,7 @@
 #include <iostream>
 #include <cmath>
 #include <vector>
+#include <unistd.h>
 
 #include "bass/bass.h"
 #include "SOIL/SOIL.h"
